docs/Events/listening-to-events.md

```diff
@@ -9,3 +9,72 @@ parent: "Events"
 ---
 
 # Listening to Events
+
+`bpmn-js` uses a general purpose [event bus](../Modules/common-modules/event-bus.md) provided by `diagram-js` to communicate across a diagram instance based on [`events`](./bpmn-events).
+
+## Retrieving the `eventBus` Instance
+
+In order to register a listener for an `event`, the `eventBus` instance needs to be retrieved first. This instance is registered as named diagram service in an internal class called `BaseViewer` and can therefore be retrieved using the `BaseViewer#get` method which is available in the sub-classes [`NavigatedViewer`](tbd), [`Viewer`](tbd) or [`Modeler`](tbd).
+
+Alternatively, the `#on` and `#off` methods can be used in [`NavigatedViewer`](tbd), [`Viewer`](tbd) or [`Modeler`](tbd) to directly register or remove a previously registered event listener.
```

Comment:
Just one additional thought: instead of directing to an empty / 404 page, we could just use

```markdown
[`Viewer`](#)
```

as the placeholder so we at least stay on the same page